prov/gni/src/gnix_cq.c

```diff
@@ -721,6 +723,12 @@ DIRECT_FN int gnix_cq_open(struct fid_domain *domain, struct fi_cq_attr *attr,
 	if (ret)
 		goto err4;
 
+	cq_priv->error_data = calloc(1, sizeof(struct gnix_ep_name));
+	if (cq_priv->error_data == NULL) {
+		ret = -FI_ENOMEM;
+		goto err4;
```

Comment:
I would like better error labels than err#. Please change the label to something more meaningful.